src/music21/base.ts

```diff
@@ -484,6 +484,20 @@ export class Music21Object extends prebase.ProtoM21Object {
             }
         }
     }
+
+    repeatAppend(object, item, numberOfTimes) {
+        let unused = null;
+        try {
+            unused = item.isStream;
+            console.log(item.isStream);
+        } catch (AttributeError) {
+            console.log('to put a non Music21Object in a stream, '
```

Comment:
The message is great, but it should throw a new StreamException -- otherwise programs that call this function cannot catch that an error has occurred.  And then `object` will still have `item` appended to it.  So a stream that expects notes might have the number `5` appended to it, which will cause problems down the road.

Unfortunately Javascript does not allow you to catch particular classes of errors, just all errors or no errors, so the `catch (AttributeError)` does nothing that `catch ()` wouldn't do the same as.